packages/itwinui-css/backstop/tests/expandable-block.html

```diff
@@ -46,7 +46,7 @@ <h1>Expandable block</h1>
     <h2>Default</h2>
     <section id="demo-default">
       <div class="iui-expandable-block">
-        <div
+        <button
```

Comment:
`.iui-header` can remain being a `<div>` with the addition of `role="button"` or it can be a `<button>`, following [this accessibility guide](https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion).

Reply:
Ohh, sounds good, just confirming, since the other examples in the file used `div` with `role="button"`.